compiler/modules/front.syntax/parseStmt.bal

```diff
@@ -218,21 +217,24 @@ function finishCallStmt(Tokenizer tok, CallStmt expr, Position startPos) returns
     return stmt;
 }
 
+// call after ingesting checkingKeyword startPos set to position of checkingKeyword
 function finishCheckingCallStmt(Tokenizer tok, CheckingKeyword checkingKeyword, Position startPos) returns CallStmt|err:Syntax {
     Token? t = tok.current();
     if t is "check"|"checkpanic" {
```

Comment:
I still don't understand this. Maybe you explained it in the old pull request.

Reply:
renamed as suggested in #663